db/migrate/20151118183909_modify_sales.rb

```diff
@@ -0,0 +1,6 @@
+class ModifySales < ActiveRecord::Migration
+  def change
+    add_index :sales, :vendor_id
```

Comment:
For all of these migrations to add indexes, we could actually have included those in the original migration for creating the model:
```ruby
def change
  create_table :sales do |t|
    t.datetime :purchase_time
    t.integer :amount
    t.integer :vendor_id, index: true
    t.integer :product_id, index: true

    t.timestamps null: false
  end
end
```